src/test/java/org/opensearch/security/test/helper/rest/RestHelper.java

```diff
@@ -433,6 +434,22 @@ public boolean isJsonContentType() {
             return ct.contains("application/json");
         }
 
+        public String getTextFromJsonBody(String jsonPointer) {
+            return getJsonNodeAt(jsonPointer).asText();
+        }
+
+        private JsonNode getJsonNodeAt(String jsonPointer) {
```

Comment:
qq: If the method getJsonNodeAt(jsonPointer) does not find the specified node, what is returned? Is it null? If it's a possible scenario, we might want to handle it or at least document the expected behavior.

Reply:
I ran into this when testing to figure out the correct JSON path. It returns an empty string.

This method is used in assertion statements like this:

```
Assert.assertEquals(res.getTextFromJsonBody("/hits/total/value"), "1");
```

so the assertion will fail.